packages/widgets/src/blocks/legacy-widget/index.php

```diff
@@ -24,14 +24,23 @@ function render_block_core_legacy_widget( $attributes ) {
 		return '';
 	}
 
-	$id_base = $attributes['idBase'];
+	$id_base    = $attributes['idBase'];
+	$widget_key = '';
 	if ( method_exists( $wp_widget_factory, 'get_widget_key' ) ) {
```

Comment:
@spacedmonkey Why is this check here? This method will exist in 5.8 once [PR 1378](https://github.com/WordPress/wordpress-develop/pull/1378) gets merged.

Reply:
What if you run this plugin in WordPress 5.6/5.7?